components/ModalSheet.js

```diff
@@ -48,12 +54,15 @@ export default ({
         swipeDirection="down"
         style={styles.modal}
       >
-        <View style={styles.container}>
+        <KeyboardAvoidingView
```

Comment:
lol...idk how this was working before...
this allows the inputs to get pushed up so the keyboard doesn't cover them

Reply:
@dereckquock it wasn't working before. I had to tap on other fields to get it to scroll up for me 😄 I should've said something, but it was a nitpick.

Reply:
really? it works for me 🤔